edgelet/iotedge-proxy/src/logging.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::env;
+
+use failure::Fail;
+use log::{log, Level, LevelFilter};
+
+pub fn init() {
+    env_logger::Builder::new()
+        .filter_level(LevelFilter::Info)
```

Comment:
Should our default level be higher?

Reply:
that is what iotedged itself uses right now. we can set ENV PROXY_LOG=debug as needed